config/runtimes/kserve-huggingfaceserver.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: serving.kserve.io/v1alpha1
```

Comment:
This also needs to be added in helm charts. https://github.com/kserve/kserve/blob/master/charts/kserve-resources/templates/clusterservingruntimes.yaml